build_tools/python/e2e_model_tests/cmake_generator.py

```diff
@@ -26,6 +26,10 @@ def generate_rules() -> List[str]:
     runner_args = run_module_utils.build_run_flags_for_model(
         model=model,
         model_input_data=test_config.input_data) + test_config.extra_test_flags
+    # TODO(#11136): Currently the DRIVER is a separate field in the CMake rule (
```

Comment:
Not really clear what the TODO is here. It seems like this is already done? What would you want to change?

Reply:
I still want to try to generate the flags for tests w/o have the separate DRIVER field. Some changes on the CMake rules are required.